WDE.MapRenderer/Managers/WmoManager.cs

```diff
@@ -154,6 +157,11 @@ public IEnumerator LoadWorldMapObject(string path, TaskCompletionSource<WmoInsta
                 group.Dispose();
             }
 
+            for (var i = wmo.doodadSets[wmoObject.DoodadSet].startIndex; i < wmo.doodadSets[wmoObject.DoodadSet].count; i++)
```

Comment:
What’s the use here?

Reply:
it's an example for only loading doodads from the current wmo placement set, I commented it until actual rendering implementation